api-editor/gui/src/features/packageData/selectionView/ActionBar.tsx

```diff
@@ -114,6 +114,28 @@ export const ActionBar: React.FC<ActionBarProps> = function ({ declaration }) {
     );
 };
 
+export const getAllSelectedElements = function (current: PythonDeclaration, filter: AbstractPythonFilter, annotations: AnnotationStore, usages: UsageCountStore): PythonDeclaration[] {
```

Comment:
Can now use `useAppSelector(selectFlatSortedDeclarationList)` and filter this list. The list is cached, so performance should be a little better.